app/utils/database.py

```diff
@@ -1100,8 +1099,9 @@ def init():
     with DATABASE.atomic():
         sh_bootstrap.save()
 
-    ## Create sequence to allow unique ids for digital certificates
-    if type(DATABASE) == PostgresqlDatabase:
+    # Create sequence to allow unique ids for digital certificates
+    if isinstance(DATABASE, PostgresqlDatabase):
+        # pylint: disable=E501
```

Comment:
Out of curiosity: What does this pylint-comment do? What is being disabled?

Reply:
This was likely meant as a flake8 disablement (for long lines).